src/main/java/com/google/devtools/build/lib/bazel/rules/java/java_stub_template.txt

```diff
@@ -363,8 +363,17 @@ if [ -z "$CLASSPATH_LIMIT" ]; then
 fi
 
 if (("${#CLASSPATH}" > ${CLASSPATH_LIMIT})); then
-  export JACOCO_IS_JAR_WRAPPED=1
-  create_and_run_classpath_jar
+  java_ver=$(JAVABIN --version 2>&1) | head -1
+  if [[ "$java_ver" =~ [^.0-9]1[.][1-8] ]]; then
```

Comment:
This regex will accept versions 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8. Just want to double-check that this is the expected behavior. Do you care about version 1.0?